server-state-machine.go

```diff
@@ -785,6 +800,7 @@ type ServerStateWaitFinished struct {
 	handshakeHash       hash.Hash
 	clientTrafficSecret []byte
 	serverTrafficSecret []byte
+	exporterSecret      []byte
```

Comment:
The duplication here is starting to look pretty bad.  Any reason why these state variables can't be bound up into a struct that can be reused?  This is probably a question for @bifurcation who created this miss in the first place.